### PR TITLE
Assignment02

### PR DESCRIPTION
**Pull request by beckySchmitty**
Week 2 Homework Submission PR

### CODE REVIEW

**Comment by beckySchmitty** [comment]
@DesignationM & @dvrzan

File: week2/starter/ColorPicker/Views/ButtonViews.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import SwiftUI
+
+// Text for button used on main color picker screen
+struct ButtonText: View {
+  var text: String
+  var color: String
+
+    var body: some View {
+
```
Comment (dvrzan) [inline week2/starter/ColorPicker/Views/ButtonViews.swift]: There's an extra space here that you don't need.

File: week2/starter/ColorPicker/Views/ContentView.swift [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  @State private var foregroundColor = Color(red: 0.0, green: 0.0, blue: 0.0)
+  
+  @Environment(\.verticalSizeClass) var verticalSizeClass
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass
+  
+  var body: some View {
+
+    if (verticalSizeClass == .regular && horizontalSizeClass == .compact) {
+        // horizontal (portrait) orientation
+      ZStack {
```
Comment (dvrzan) [inline week2/starter/ColorPicker/Views/ContentView.swift]: No need for a `ZStack` here.

File: week2/starter/ColorPicker/Views/ContentView.swift [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        VStack {
+          Text("Color Picker")
+            .font(.largeTitle)
+            .foregroundColor(Color("TextColor"))
+            .bold()
+          RoundedRectangle(cornerRadius: 0)
+            .foregroundColor(foregroundColor)
+            .border(.black)
+            .overlay(
```
Comment (dvrzan) [inline week2/starter/ColorPicker/Views/ContentView.swift:57]: You already have the border above so no need to add an overlay as well. Just add the color to the border and apply the width :]

File: week2/starter/ColorPicker/Views/SliderViews.swift [hunk 4]
```diff
@@ -32,57 +32,32 @@
...
 
-struct ContentView: View {
-  @State private var alertIsVisible: Bool = false
-  @State private var redColor: Double = 0.0
-  @State private var greenColor: Double = 0.0
-  @State private var blueColor: Double = 0.0
-  @State private var foregroundColor = Color(red: 0, green: 0, blue: 0)
-
-  var body: some View {
+struct colorSlider: View {
```
Comment (dvrzan) [inline week2/starter/ColorPicker/Views/SliderViews.swift]: struct should be uppercased -> `ColorSlider`

**Review by dvrzan — APPROVED** [review]
Nice work on your homework!

**Pull request merged**